source/expressions.tex

```diff
@@ -4689,6 +4684,9 @@
 the coefficient $\tcode{r}_i$
 of the base-2 representation of the result \tcode{r}
 is 1 if $\tcode{x}_i$ is 0, and 0 otherwise.
+\begin{note}
+This operation is also called ``unary complement'' or ``ones' complement''.
+\end{note}
```

Comment:
@burblebee, @jensmaurer, @opensdh: could you please weigh in on whether such an explanatory note is appropriate? We use the term "unary complement" normatively in the next sentence, and it hasn't been defined before. It's "common knowledge" I suppose, which is why I just made a note, not a definition.

Reply:
I think "unary complement" is referring to the nature of the operator, not the operation.  We should just follow the style of [expr.bit.and] _et al._:
> The result is the ones' complement function of the operand.

Reply:
How about we say both the term and the spelling-out:

> <ins>The result is the ones' complement of the promoted operand, that is, </ins>given the coefficients [...]

Reply:
I don't see a reason to be inconsistent with the other bitwise operators, but it's not harmful.  (It would be "operand:", though.)

Reply:
I see. Hm, the `~` wording is in a different place and context from the `&&` wording, but we can add that note. I suppose we should also add a note to the two's complement (`-`)?

Reply:
@opensdh: Please have a look, I added notes for both ones' and two's complement.

Reply:
I would prefer if we could apply this core issue as-is and move any additional editorial clarification deemed necessary to a separate issue.